_data/events_gdcr2018/Mumbai.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "GDCR18@Mumbai",
+  "url": "https://goo.gl/yUA1sX",
+  "moderators": [
+    "@vedantseta",
+    "@sandeepshetty"
+  ],
+  "location": {
+    "city": "Mumbai City",
+    "country": "India",
+    "coordinates": {
+      "latitude": 19.124681,
+      "longitude": 72.8463366
+    }
```

Comment:
Either utcOffset or timezone (preferably timezone) is required so we can match you to a video room at the day of the event. Please see to the other events for an example